Dockerfile

```
@@ -1,16 +1,16 @@
 FROM golang:1.13 as bd
 RUN adduser --disabled-login appuser
-WORKDIR /github.com/layer5io/meshery-maesh
+WORKDIR /github.com/layer5io/meshery-traefik-mesh
 ADD . .
-RUN GOPROXY=direct GOSUMDB=off go build -ldflags="-w -s" -a -o /meshery-maesh .
-RUN find . -name "*.go" -type f -delete; mv maesh /
+RUN GOPROXY=direct GOSUMDB=off go build -ldflags="-w -s" -a -o /meshery-traefik-mesh .
```

Right, here is the suggestion:

```suggestion
RUN GOPROXY=https://proxy.golang.org GOSUMDB=off go build -ldflags="-w -s" -a -o /meshery-traefik-mesh .
```

---

thank you, thank you